easybuild/easyconfigs/c/CGAL/CGAL-4.11-foss-2017b.eb

```diff
@@ -0,0 +1,40 @@
+name = 'CGAL'
+version = '4.11'
+
+homepage = 'http://www.cgal.org/'
+description = """The goal of the CGAL Open Source Project is to provide easy access to efficient
+ and reliable geometric algorithms in the form of a C++ library."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'strict': True}
+
+source_urls = ['https://github.com/CGAL/cgal/releases/download/releases%2FCGAL-%(version)s']
+sources = [SOURCE_TAR_XZ]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('Boost', '1.65.1'),
+    ('MPFR', '3.1.6'),
+    ('GMP', '6.1.2'),
+    ('libGLU', '9.0.0'),
+]
+
+builddependencies = [
+    ('CMake', '3.9.4'),
+    ('Eigen', '3.3.4'),
+    ('Qt5', '5.9.2'),
+]
+checksums = ['27a7762e5430f5392a1fe12a3a4abdfe667605c40224de1c6599f49d66cfbdd2']  # CGAL-4.11.tar.xz
```

Comment:
please move this up, right below `sources` (matter of style, order barely matters in easyconfig files)